src/main/webapp/auth.js

```diff
@@ -60,7 +60,7 @@ function handleAuthenticationState() {
     populateGmail();
     populateTasks();
     populateCalendar();
-    postAndGetTaskList();
+    // postAndGetTaskList();
```

Comment:
Did you intend to leave this commented out? Why?

Reply:
Echoing our offline conversation - this function adds tasks to a user's Tasks account on every run of the project. This change has been moved to PR #120.

Reply:
`RESOLVED`